server/index.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+const app = express();
+
+const bodyParser = require('body-parser');
+const userEndpoint = require('./controllers/userControlers').default;
+
+app.use(bodyParser.urlencoded({ extended: false}));
+app.use(bodyParser.json());
+
+// const router = require('./routes/routes')
+
+
+app.get('/api/v/anouncement', (req,res) => {
```

Comment:
`/api/v/anouncement`, `v` is not a proper way to version your API. It should be `v1`.